### DISCUSSION TITLE
Using egjs-view360 with Wordpress or another self-hosted CMS?

### DISCUSSION THREAD

**Discussion by neuhaus** [naver/egjs-view360#421] [Q&A]
First of all, egjs-view360 looks amazing! It's just what i was looking for.

I'd like to use egjs-view360 for a travel blog. To make the blog easy to edit, i want to use something like ghost or wordpress.

What's the best or easiest way to integrate egjs-view360 with such a CMS? Are there examples? I'm sure it's been done before.

Thanks!

**Comment by neuhaus** [comment]
Is noone using egjs-view360 for a blog?

**Comment by daybrush** [comment]
@neuhaus 

hi.

The injection method for medium or CMS is not supported.
You have to create a plug-in yourself or run a script.

https://naver.github.io/egjs-view360/docs

**Comment by malangfox** [comment]
Since there is no further comment, we will close the discussion.
